<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_memeber</class>
 <widget class="QDialog" name="edit_memeber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbSignUpText">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Text*/

position: absolute;
width: 164px;
height: 56px;
left: 112px;
top: 47px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 36px;
line-height: 56px;
display: flex;
align-items: center;
text-align: center;</string>
   </property>
   <property name="text">
    <string>Edit Books</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBorder1">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>70</y>
     <width>741</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Rectangle 1 */


position: absolute;
width: 794px;
height: 491px;
left: 34px;
top: 112px;

background: #C4C4C4;
border-radius: 8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEditMeb">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>92</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Border 2 */


position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 8px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelEdMebmberNTitle">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>92</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Name: */


position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEdUserNameText">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Name: */


position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEditUser">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Border 2 */


position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 8px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelEdPasswordText">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Name: */


position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEditPass">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>190</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Border 2 */


position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 8px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEditEmail">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Border 2 */


position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 8px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelEdEmailText">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>240</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Name: */


position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEdBirthdayText">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>290</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*birthday text*/

position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Birthday:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEdGenderText">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>292</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Gender Text*/

position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="EdtSelectDate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*boader Date*/
position: absolute;
width: 156px;
height: 37px;
left: 239px;
top: 352px;

background: #EDEDED;
border-radius: 8px; 

/*Text Date*/ 

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;</string>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cBxEdGen">
   <property name="geometry">
    <rect>
     <x>572</x>
     <y>292</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*boader Gender*/
position: absolute;
width: 156px;
height: 37px;
left: 239px;
top: 352px;

background: #EDEDED;
border-radius: 8px; 

/*Text Gender*/ 

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEdAdressText">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>344</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Address Text*/

position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Address:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEdAdd">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>346</y>
     <width>641</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Border Sign-up 4 */

position: absolute;
width: 548px;
height: 37px;
left: 202px;
top: 128px;

background: #EDEDED;
border-radius: 8px; 

/*Text Email*/ 

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;s</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sBEdChildrenNo">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Children spin Box*/

position: absolute;
width: 99px;
height: 37px;
left: 239px;
top: 542px;

background: #EDEDED;
border-radius: 10px; 

/*Number Text*/ 

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="chB_EditEmails">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>420</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 40px;
height: 40px;
left: 627px;
top: 545px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelEDRecieveEmailText">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>410</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Address Text*/

position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Recieve Emails:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEdChildrenText">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>410</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Address Text*/

position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Children:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEditSave">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Border Sign-up 4 */

position: absolute;
width: 548px;
height: 37px;
left: 202px;
top: 128px;

background: #EDEDED;
border-radius: 10px; 

/*Text Email*/ 

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 20px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;
</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
